GoogleCloudPlatform/microservices-demo
# Support for Tilt.dev

### Background 
Support for Tilt.dev

### Fixes 
Faster application development in a local Kubernetes environment

### Change Summary
- Add Tiltfile
- Added to `docs/development-guide.md` information on how to run Tilt locally along with sample screenshots

### Testing Procedure
- Install tilt.dev
- Required Docker Desktop with enabled single-node Kubernetes cluster
- clone this repository
- run `tilt up`
- open tilt dashboard `http://localhost:10350` to check deployed services or run drom command line `kubectl get pods`
